src/lib/storage/table.cpp

```diff
@@ -32,6 +32,18 @@ Table::Table(const TableColumnDefinitions& column_definitions, const TableType t
   DebugAssert(!max_chunk_size || *max_chunk_size > 0, "Table must have a chunk size greater than 0.");
 }
 
+Table::Table(const TableColumnDefinitions& column_definitions, const TableType type, std::vector<std::shared_ptr<Chunk>>&& chunks, const UseMvcc use_mvcc):
+  Table(column_definitions, type, type == TableType::Data ? std::optional{Chunk::MAX_SIZE} : std::nullopt, use_mvcc) {
+
+  _chunks = {chunks.begin(), chunks.end()};
+
+#if HYRISE_DEBUG
+  for (const auto& chunk : _chunks) {
+    DebugAssert(chunk->has_mvcc_data() == (_use_mvcc == UseMvcc::Yes), "Supply MvccData for Chunks iff Table uses MVCC");
```

Comment:
For consistency, either use only DebugAssert (here) or only Assert (chunk.cpp) within `HYRISE_DEBUG`.

Reply:
Going through Hyrise's codebase there is no consensus whether DebugAssert or Assert should be used in #if HYRISE_DEBUG and I don't think it's worth establishing one. I only added #if HYRISE_DEBUG here to avoid potential "empty loop!!!!" warnings in release builds and to make obvious that the entire loop is only meant for debug builds